seannkelleyy/Mario-Remake
# IH-add-full-screen

﻿
# Description

New feat: Added full screen capability
Bug Fix: Fixed an issue with the bullet bill that caused a build error.

closes #144 

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

# How To Test

Just run the game and make sure the aspect ratio is good on your devices that have different size screens.


# Checklist:
